src/components/Content/Content.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { useEffect, useState } from 'react'
+import Home from '../Pages/Home'
+
+type ContentProps = {
+    renderedContent: React.ReactNode; // Specify the type of the renderedContent prop
+};
+
+function Content({ renderedContent }: ContentProps) {
+    return (
+        <div>
+            {renderedContent} {/* This will render the passed ReactNode */}
+        </div>
```

Comment:
Replace div with empty element:

```suggestion
        <>
            {renderedContent} {/* This will render the passed ReactNode */}
        </>
```